examples/examples.py

```diff
@@ -107,7 +107,7 @@
     data_format = "local_db"  # "local_db" or "timescale"
 
     # select the example to run from the available examples above
-    example = "small"
+    example = "small_learning_1"
```

Comment:
this should not change?

Reply:
> the other two commits are used to reproduce the mentioned issue

Yes, I will revert those, once we know whats up with #512